cpp/src/barretenberg/benchmark/honk_bench/compare_honk_branch_vs_baseline.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
If you start with your branch you might be able to save some compilation time

Reply:
Good point. It's currently set up to nuke the build dir anyway to be extra careful but I've gone ahead and reversed the order so that if someone using the script wants to save time its as easy as commenting that line out.